tests/common/file_utils_test.py

```diff
@@ -306,6 +306,22 @@ def test_cached_path(self):
         with open(filename, "r") as f:
             assert f.read().startswith("I mean, ")
 
+    @responses.activate
+    def test_cached_path_http_err_handling(self):
+        url_404 = "http://fake.datastore.com/does-not-exist"
```

Comment:
Can we use a URL that 404s more reliably? Maybe point to our own site, or to GitHub? I'm a little worried that someone at datastore.com can cause our tests to fail.

Reply:
With the `@responses.activate` decorator, external requests are never made. Every URL you hit has to be mocked.

Reply:
Oh, sweet!